docs/release-notes/index.md

```diff
@@ -21,8 +21,157 @@ To check for security updates, go to [Security announcements for the Elastic sta
+* Adds the `MIGRATE` action to the `SIGNED_ACTIONS` set [#228566]({{kib-pull}}228566).
+* Adds a **Show agentless resources** toggle on the Fleet > Settings page to allow for debugging and diagnostics [#237528]({{kib-pull}}237528).
+* Updates the maximum supported version of package spec to 3.5 [#235942]({{kib-pull}}235942).
+* Enables the installation of `alerting_rule_template` {{kib}} assets from packages; also enables a background task for reporting agent status changes in {{fleet}} [#235842]({{kib-pull}}235842).
+* Adds a filter on the Integrations page to show only agentless integrations [#235686]({{kib-pull}}235686).
+* Adds support for a new `url` variable type in {{fleet}} packages which provides better input validation of URLs in configurations [#231062]({{kib-pull}}231062).
+* Adds support for a new `duration` variable type in {{fleet}} packages which provides better input validation of duration strings in configurations [#231027]({{kib-pull}}231027).
+* Renders the accordion for sample events on the integration overview page as collapsed by default [#228799]({{kib-pull}}228799).
+* Hides metrics data streams and packages on Logs Essentials projects [#227842]({{kib-pull}}227842).
+* Adds a package rollback API [#226754]({{kib-pull}}226754).
+* Adds support for the `keep` option in the Remove processor in ingest pipelines [#225638]({{kib-pull}}225638).
+* Adds the **Add / remove tags** action on the agent details page [#225433]({{kib-pull}}225433).
+
+**Discover**:
+* You can now manage multiple data explorations simultaneously in Discover using tabs. Navigate between them seamlessly, duplicate them to build on previous queries, and save them into a single Discover session to retrieve your queries, filters, and data configurations untouched at a later time [#235150]({{kib-pull}}235150).
+* You can now send long-running queries to run in the background. When a query runs in the background, you can perform other queries and tasks, then open it later from a new Background searches flyout to view cached results. This feature is in technical preview and disabled by default in this release. Check [this page](docs-content://explore-analyze/discover/background-search.md) for instructions [#236818]({{kib-pull}}236818).
+* Improves Discover’s {{esql}} LOOKUP JOIN experience with an index editor flyout supporting inline editing, CSV import, contextual actions based on privileges, and a query bar that opens Discover with a prefilled ES|QL query [#232686]({{kib-pull}}232686).
+* You can now add controls directly in Discover by typing variables (for example `?service` or `??host.name`) in your {{esql}} queries. Instead of having to maintain multiple similar queries, controls let you dynamically change parts of a single query using dropdowns, multi-selects, and date pickers with value suggestions [#229598]({{kib-pull}}229598).
```

Comment:
Yes this is not 100% correct. 

> Instead of having to maintain multiple similar queries, controls let you dynamically change parts of a single query using dropdowns, multi-selects, and date pickers with value suggestions

This is the most confusing part. In controls you create an ES|QL variable (?service for values ??field for fields) and then you associate it with a dropdown. We don't have multi selects or date pickers

Can we change it in order to be closer to what we have? Otherwise it will be confusing. Also does it make sense to say something like: exactly as it works in dashboards (as it is the same feature)